### PR TITLE
BZ1853309: removed unhelpful link and redirected customers to the location of the image needed for their release

### PR DESCRIPTION
**Pull request by skrthomas**
For versions 4.5+

https://bugzilla.redhat.com/show_bug.cgi?id=1853309

Direct link to doc preview: https://deploy-preview-30123--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-restricted-networks-vsphere.html#installation-vsphere-machines_installing-restricted-networks-vsphere

Ready for QA: @wjiangjay

### CODE REVIEW

File: modules/installation-vsphere-machines.adoc [hunk 1]
```diff
@@ -42,8 +42,8 @@ If you plan to add more compute machines to your cluster after you finish instal
 ====
 
 ifndef::openshift-origin[]
-. Obtain the {op-system} OVA image from the link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red Hat Customer Portal or the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/[{op-system} image mirror] page.
-+
+. Obtain the {op-system} OVA image version that matches your {product-title} version if it is available. Images are available from the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/[{op-system} image mirror] page, under `your-openshift-container-platform-version/latest/`
```
Comment (bmcelvee) [inline modules/installation-vsphere-machines.adoc:45]: Suggested edit to match the file name: 

s/ `your-openshift-container-platform-version/latest/` / `/pub/openshift-v4/dependencies/<op-system>/latest/`.

**Comment by bmcelvee** [comment]
Suggested one edit, otherwise LGTM!

**Comment by jinyunma** [comment]
LGTM for the update.
I have some other comments:
1. I checked that other platform installation docs( gcp/baremetal...) also have similar description about how to get rhcos image. Do you plan to update them together?
2. The line after updated line describes the file name, the format seems not matched with download page.
rhcos-<version>-vmware.<architecture>.ova  -> rhcos-vmware.<architecture>.ova

**Comment by skrthomas** [comment]
@jinyunma Thanks for your feedback! I think I tracked down all the other instances of where these links occur and replicated the updates that you approved. I also addressed the file name description discrepancy that you pointed out.

File: modules/installation-gcp-user-infra-rhcos.adoc [hunk 2]
```diff
@@ -12,10 +12,8 @@ your {product-title} nodes.
 
 ifndef::openshift-origin[]
 . Obtain the {op-system} image from the
-link:https://access.redhat.com/downloads/content/290[Product Downloads] page on the Red
-Hat customer portal or the
 link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/[{op-system} image mirror]
-page.
+page, under /pub/openshift-v4/dependencies/<op-system>/latest/.
```
Comment (kalexand-rh) [inline modules/installation-gcp-user-infra-rhcos.adoc:16]: I don't think this makes sense. This text doesn't match the heading on the page, */pub/openshift-v4/dependencies/rhcos* or the link that you'd click. I think you should include /latest/ in the URL for the image mirror.

Comment (skrthomas) [reply modules/installation-gcp-user-infra-rhcos.adoc:16]: @kalexand-rh Okay, yeah I see that. I think this needed to be changed in the other files too, so I went and made sure that /latest/ was added to the URL for image mirror

Comment (kalexand-rh) [reply modules/installation-gcp-user-infra-rhcos.adoc:16]: Great! With that change, I think this makes more sense:
```suggestion
page.
```

**Pull request merged**